.github/workflows/convert.yaml

```diff
@@ -0,0 +1,56 @@
+name: Convert definitions
+
+on:
+  push:
+    branches: ["main"]
+jobs:
+  Test:
```

Comment:
This seems to be a separate job in V-F definitions and this repo, whereas the content is encapsulated in the Convert job in the sandbox definitions. Why the difference? Unless there is any good reason for a such difference, I'd want to see it consistent across all 3 repos (and the best one used, whichever it is).

Reply:
Agreed on Slack we'll go with the approach from this repo.